tests/integration/test_migrate.sh

```diff
@@ -25,7 +25,12 @@ if [ "$init" != "systemd" ];then
     exit 0
 fi
 
-dockerPid=$(ps -C docker -o pid=|xargs)
+if ! systemctl is-active docker >/dev/null; then
+     echo "Docker daemon is not running"
+     exit 1
```

Comment:
`test_migrate.sh` is already skipping the test. This code fixes a problem in capturing `docker PID` the right way, just in case someone decides to run the test.